build_backend/test_onbuild.py

```diff
@@ -38,7 +39,13 @@ def build(request: pytest.FixtureRequest, tmp_path: Path, template_fields: dict)
     (pkg_dir / "streamlink").mkdir(parents=True)
     shutil.copy(PROJECT_ROOT / "pyproject.toml", tmp_path / "pyproject.toml")
     shutil.copy(PROJECT_ROOT / "setup.py", tmp_path / "setup.py")
-    shutil.copy(PROJECT_ROOT / "src" / "streamlink" / "_version.py", pkg_dir / "streamlink" / "_version.py")
+
+    # Lookup for the path from the root source directory
+    if os.path.isfile(PROJECT_ROOT / "src" / "streamlink" / "_version.py"):
```

Comment:
`pathlib`'s equivalent of `os.path.isfile()` doesn't require an additional import and is simpler:
https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

Reply:
Thanks, I've replaced it with Path.exists() instead of is_file(), it is more in line with what I wanted to do.